zephyr/lib/cpu.c

```diff
@@ -46,6 +46,7 @@ extern struct tr_ctx zephyr_tr;
 extern void *global_imr_ram_storage;
 #endif
 
+#if CONFIG_PM
```

Comment:
Please submit this small, generic commit in a separate PR for Continuous Integration purposes.

Reply:
Done it https://github.com/thesofproject/sof/pull/9203